CONTRIBUTING.md

```diff
@@ -64,3 +64,8 @@ Build and test on changes:
 ```
 make watch
 ```
+
+Build the snapshot for testing project generation:
+```
+make build-snapshot
+```
```

Comment:
It's not immediately obvious to me how to do anything besides testing the fixtures - can we get some docs in the README on how to use this?

Reply:
Sure, and currently the only test is a diff on the fixture.